<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/0102.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>430</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/0121.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSpinBox" name="sp_infrared">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>440</y>
     <width>59</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>440</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>图表</string>
   </property>
  </widget>
  <widget class="QPushButton" name="icon_curtain">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/0113.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_curtain">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>窗帘打开</string>
   </property>
  </widget>
  <widget class="QPushButton" name="icon_tv">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/0115.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_tv">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>电视机打开</string>
   </property>
  </widget>
  <widget class="QPushButton" name="icon_dvd">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>120</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/0117.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_dvd">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>130</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>DVD打开</string>
   </property>
  </widget>
  <widget class="QPushButton" name="icon_fan">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>20</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/0119.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_fan">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>30</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>风扇打开</string>
   </property>
  </widget>
  <widget class="QPushButton" name="icon_lamp">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>70</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/0125.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_air">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>130</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>空调打开</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_lamp">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>80</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>射灯打开</string>
   </property>
  </widget>
  <widget class="QPushButton" name="icon_air">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>120</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/0122.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_warning">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>20</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/0112.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_rfid">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>80</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/0124.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>100</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>执行模式</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmb_zmode">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>130</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>模式1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>模式2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>模式3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>模式4</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="chk_mode">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>170</y>
     <width>98</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>开启</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_8">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>260</y>
     <width>87</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>条件</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="ld2">
      <item>
       <property name="text">
        <string>&gt;</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>&lt;</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_9">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>330</y>
     <width>87</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>器件</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="ld4">
      <item>
       <property name="text">
        <string>报警灯</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>风扇</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>射灯</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>窗帘</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_10">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>330</y>
     <width>87</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>动作</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="ld5">
      <item>
       <property name="text">
        <string>开</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>关</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>210</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_clear">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>210</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>727</x>
     <y>440</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/0127.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>581</x>
     <y>31</y>
     <width>83</width>
     <height>19</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <item>
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>人体：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lab_inf">
      <property name="text">
       <string>无人</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>190</y>
     <width>71</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>设置模式</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cmb_smode">
      <item>
       <property name="text">
        <string>模式1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>模式2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>模式3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>模式4</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>260</y>
     <width>87</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>传感器</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="ld1">
      <item>
       <property name="text">
        <string>温度</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>光照</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>260</y>
     <width>69</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>设置值</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="ld3"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>188</width>
     <height>308</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 255);</string>
        </property>
        <property name="text">
         <string>CO2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcd_co2">
        <property name="numDigits">
         <number>7</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 255);</string>
        </property>
        <property name="text">
         <string>GAS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcd_gas">
        <property name="numDigits">
         <number>7</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 255);</string>
        </property>
        <property name="text">
         <string>HUMI</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcd_hum">
        <property name="numDigits">
         <number>7</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 255);</string>
        </property>
        <property name="text">
         <string>LIGHT</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcd_ill">
        <property name="numDigits">
         <number>7</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 255);</string>
        </property>
        <property name="text">
         <string>PM25</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcd_pm">
        <property name="numDigits">
         <number>7</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 255);</string>
        </property>
        <property name="text">
         <string>AIR</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcd_air">
        <property name="numDigits">
         <number>7</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 255);</string>
        </property>
        <property name="text">
         <string>SMOKE</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcd_smoke">
        <property name="numDigits">
         <number>7</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 255);</string>
        </property>
        <property name="text">
         <string>TEMP</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcd_temp">
        <property name="numDigits">
         <number>7</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
